### ISSUE TITLE
Error spamm / Sandbox

### ISSUE THREAD

**Issue by deinemudda32** [Facepunch/garrysmod-issues#1804]
[ERROR] lua/matproxy/player_color.lua:28: Tried to use a NULL entity!
  1. GetPlayerColor - [C]:-1
   2. bind - lua/matproxy/player_color.lua:28
    3. unknown - lua/includes/modules/matproxy.lua:56

**Comment by robotboy655** [comment]
You will need to provide way more info than this. Theoretically this error cannot happen.

You'll need to provide info such as:
* Gamemode
* When exactly this occurs, at which circumstances
* What version of the game are you running

**Comment by deinemudda32** [comment]
Gamemode: Sandbox
Occur: I used to watch this error for now 1 year. I didnt get any solution. 
Some say its a conflict between the Slig Wolfs Train.
Some say it happens when you spawn hunter plate 7x7.
And some say it happens if you ulx ban them.

Version: newest version of the game. Previously I changed from lua format to addon format to get it updated but I used to get this error since 1 year.
Just add a valid check or somthin I dont know exactly. I removed some lines to get it work but then players can't choose a color for themself anymore. thats the problem. :(
